### PR TITLE
Add link to single view in map info bubble

### PR DESCRIPTION
**Pull request by heikohaensge**
Add a link to the single view page if the custom typoscript setup variable settings.map.singlePid is set. 
Please see extended typoscript in setup,typoscript as well. 
Actually id would be best if the already existing typoscript setup variable settings.singlePid and the according value in the plugin would be considered for this to work, but this value doesn't work (yet) in the partial Maps.html

### CODE REVIEW

**Comment by BastianBalthasarBux** [comment]
Will be added, but slightly different (settings and css-class), see my referenced commit.

File: Resources/Private/Partials/Maps.html [hunk 1]
```diff
@@ -59,6 +59,12 @@ <h6>{address.fullName}</h6>
                         <div>
                             <f:render partial="Address" section="address" arguments="{_all}"/>
                             <f:render partial="Address" section="contact" arguments="{_all}"/>
+                            <f:if condition="{settings.map.singlePid}">
+                                <f:link.action action="show" arguments="{address : address}"
+							        title="{address.title} {address.fullName}"
```
Comment (georgringer) [inline Resources/Private/Partials/Maps.html:64]: `{address.fullName}` already contains the title, no need to add it twice

File: Resources/Private/Partials/Maps.html [hunk 2]
```diff
@@ -59,6 +59,12 @@ <h6>{address.fullName}</h6>
...
+                                    pageUid="{settings.map.singlePid}" class="btn">{f:translate(key:'more')} &hellip;
```
Comment (georgringer) [inline Resources/Private/Partials/Maps.html:65]: I would remove the `&hellip;`, there is no need for that in the button

Comment (BastianBalthasarBux) [reply Resources/Private/Partials/Maps.html:65]: I would remove class="btn" too. No need for it, link is IMHO sufficient. Isn't it?

**Pull request closed without merging**